### DISCUSSION TITLE
Switch off RATTLE

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker-gpu#240] [Q&A]
Hi,

what would be the correct way of switching off RATTLE completely, so that a totally unconstrained MD is performed. The user guide says "In the absence of any modifier, RATTLE constrains all bonds to hydrogen atoms at ideal bond lengths.", but doesn't specify if something like "RATTLE none" works. Thx in advance!

Best,
Jacek

**Comment by zhi-wang** [comment]
I think if "RATTLE" and "RATTLE WATER" do not appear in the key file, the system is assumed to be flexible.

**Reply by jacekkozuch** [reply]
Ah, ok thx! I think I misread as 'if RATTLE is not specified', but it says 'if no modifier'. Thanks for clarifying!
